src/index.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
         input = String(input)
         // Reset Enquirer's styles
         let options = {
-          prefix: colors.white(colors.symbols.question),
+          prefix: colors.white(colors.symbols?.question ?? '?'),
```

Comment:
Looks like per the library we're calling here on Windows it would be `questionSmall`, maybe instead do `colors.symbols.question || colors.symbols.questionSmall`?

Reply:
Hey Ryan, so yep it is on windows. Here is a screenshot of the error when I run it locally:
![image](https://user-images.githubusercontent.com/11023698/218175816-343521a5-6295-49d2-aa0a-ea0d14f10b7e.png)

It seems like symbols is not defined, so I am afraid that questionSmall would give the same errors

I followed the steps on the npm page, and created a simple sandbox-invoke-mocks.js with the a single mock under events:
```
module.exports = {
  events: {
    'background-task': {
      'my-first-mock': { /* payload */ },
    }
  },
}
```

I don't know if it is maybe a use case thing, or maybe I have some other packages that are causing conflicts locally?

Reply:
I also see that Gulp is using the ansi-colors as well, but they are at version 1.1.0
![image](https://user-images.githubusercontent.com/11023698/218177496-8e77f2fa-de7f-41aa-a3f6-a4b3ff18812e.png)

Reply:
Very weird and unfortunate dependency resolution going on there. Will get this merged in with https://github.com/architect/plugin-lambda-invoker/pull/3 for the 1.0 release today.